### ISSUE TITLE
Error while processing RsAssembly

### ISSUE THREAD

**Issue by Painstouch** [bdew-minecraft/gendustry#61]
Might be due to updating BC to 6.1.4, but regardless posting this here for future compatibility!

```
[11:45:06] [Client thread/ERROR] [bdlib]: Error while processing RsAssembly(List((D,2), (Y,1), (Q,2)),100000,StackItem(gendustry,GeneticsProcessor,32767),1)
java.lang.NoSuchMethodError: buildcraft.core.recipes.AssemblyRecipeManager.getRecipes()Ljava/util/List;
at net.bdew.gendustry.config.loader.Loader.processRecipeStatement(Loader.scala:80) ~[Loader.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:389) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:387) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at scala.collection.immutable.List.foreach(List.scala:383) [List.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatementsSafe(RecipeLoader.scala:387) [RecipeLoader.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatementsInSubcontext(RecipeLoader.scala:366) [RecipeLoader.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatement(RecipeLoader.scala:335) [RecipeLoader.class:?]
at net.bdew.gendustry.config.loader.Loader.processRecipeStatement(Loader.scala:109) [Loader.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:389) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:387) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at scala.collection.immutable.List.foreach(List.scala:383) [List.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatementsSafe(RecipeLoader.scala:387) [RecipeLoader.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatementsInSubcontext(RecipeLoader.scala:366) [RecipeLoader.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatement(RecipeLoader.scala:342) [RecipeLoader.class:?]
at net.bdew.gendustry.config.loader.Loader.processRecipeStatement(Loader.scala:109) [Loader.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:389) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:387) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at scala.collection.immutable.List.foreach(List.scala:383) [List.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatementsSafe(RecipeLoader.scala:387) [RecipeLoader.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatements(RecipeLoader.scala:378) [RecipeLoader.class:?]
at net.bdew.gendustry.config.loader.TuningLoader$.loadDealayed(TuningLoader.scala:20) [TuningLoader$.class:?]
at net.bdew.gendustry.Gendustry$.init(Gendustry.scala:85) [Gendustry$.class:?]
at net.bdew.gendustry.Gendustry.init(Gendustry.scala) [Gendustry.class:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0_67]
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:513) [FMLModContainer.class:?]
at sun.reflect.GeneratedMethodAccessor7.invoke(Unknown Source) ~[?:?]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) [guava-16.0.jar:?]
at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.post(EventBus.java:275) [guava-16.0.jar:?]
at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:208) [LoadController.class:?]
at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source) ~[?:?]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at com.creativemd.minecraftloader.MinecraftDummyContainer.callMethod(MinecraftDummyContainer.java:209) [MinecraftDummyContainer.class:?]
at com.creativemd.minecraftloader.MinecraftDummyContainer.onModEvent(MinecraftDummyContainer.java:201) [MinecraftDummyContainer.class:?]
at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java) [LoadController.class:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0_67]
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) [guava-16.0.jar:?]
at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.post(EventBus.java:275) [guava-16.0.jar:?]
at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:118) [LoadController.class:?]
at cpw.mods.fml.common.Loader.initializeMods(Loader.java:691) [Loader.class:?]
at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:288) [FMLClientHandler.class:?]
at com.creativemd.minecraftloader.ClientHelper.startGame(ClientHelper.java:307) [ClientHelper.class:?]
at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java) [bao.class:?]
at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:867) [bao.class:?]
at net.minecraft.client.main.Main.main(SourceFile:148) [Main.class:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0_67]
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.11.jar:?]
at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.11.jar:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0_67]
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:286) [NewLaunch.jar:?]
at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:376) [NewLaunch.jar:?]
at org.multimc.EntryPoint.listen(EntryPoint.java:165) [NewLaunch.jar:?]
at org.multimc.EntryPoint.main(EntryPoint.java:54) [NewLaunch.jar:?]
[11:45:06] [Client thread/INFO] [gendustry]: Adding assembly recipe: List((D,2), (Z,1), (L,2)) + 150000 mj => StackItem(gendustry,EnvProcessor,32767) * 1
[11:45:06] [Client thread/INFO] [gendustry]: meta/damage is unset in StackItem(minecraft,diamond,32767), defaulting to 0
[11:45:06] [Client thread/INFO] [gendustry]: D -> 2xitem.diamond@0
[11:45:06] [Client thread/INFO] [gendustry]: Z -> 1xitem.redstoneChipset@2
[11:45:06] [Client thread/INFO] [gendustry]: L -> 2xitem.dyePowder@4
[11:45:06] [Client thread/INFO] [gendustry]: Output: 1xitem.gendustry.EnvProcessor@32767
[11:45:06] [Client thread/ERROR] [bdlib]: Error while processing RsAssembly(List((D,2), (Z,1), (L,2)),150000,StackItem(gendustry,EnvProcessor,32767),1)
java.lang.NoSuchMethodError: buildcraft.core.recipes.AssemblyRecipeManager.getRecipes()Ljava/util/List;
at net.bdew.gendustry.config.loader.Loader.processRecipeStatement(Loader.scala:80) ~[Loader.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:389) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:387) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at scala.collection.immutable.List.foreach(List.scala:383) [List.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatementsSafe(RecipeLoader.scala:387) [RecipeLoader.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatementsInSubcontext(RecipeLoader.scala:366) [RecipeLoader.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatement(RecipeLoader.scala:335) [RecipeLoader.class:?]
at net.bdew.gendustry.config.loader.Loader.processRecipeStatement(Loader.scala:109) [Loader.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:389) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:387) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at scala.collection.immutable.List.foreach(List.scala:383) [List.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatementsSafe(RecipeLoader.scala:387) [RecipeLoader.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatementsInSubcontext(RecipeLoader.scala:366) [RecipeLoader.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatement(RecipeLoader.scala:342) [RecipeLoader.class:?]
at net.bdew.gendustry.config.loader.Loader.processRecipeStatement(Loader.scala:109) [Loader.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:389) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at net.bdew.lib.recipes.RecipeLoader$$anonfun$processRecipeStatementsSafe$1.apply(RecipeLoader.scala:387) [RecipeLoader$$anonfun$processRecipeStatementsSafe$1.class:?]
at scala.collection.immutable.List.foreach(List.scala:383) [List.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatementsSafe(RecipeLoader.scala:387) [RecipeLoader.class:?]
at net.bdew.lib.recipes.RecipeLoader.processRecipeStatements(RecipeLoader.scala:378) [RecipeLoader.class:?]
at net.bdew.gendustry.config.loader.TuningLoader$.loadDealayed(TuningLoader.scala:20) [TuningLoader$.class:?]
at net.bdew.gendustry.Gendustry$.init(Gendustry.scala:85) [Gendustry$.class:?]
at net.bdew.gendustry.Gendustry.init(Gendustry.scala) [Gendustry.class:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0_67]
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:513) [FMLModContainer.class:?]
at sun.reflect.GeneratedMethodAccessor7.invoke(Unknown Source) ~[?:?]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) [guava-16.0.jar:?]
at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.post(EventBus.java:275) [guava-16.0.jar:?]
at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:208) [LoadController.class:?]
at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source) ~[?:?]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at com.creativemd.minecraftloader.MinecraftDummyContainer.callMethod(MinecraftDummyContainer.java:209) [MinecraftDummyContainer.class:?]
at com.creativemd.minecraftloader.MinecraftDummyContainer.onModEvent(MinecraftDummyContainer.java:201) [MinecraftDummyContainer.class:?]
at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java) [LoadController.class:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0_67]
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) [guava-16.0.jar:?]
at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) [guava-16.0.jar:?]
at com.google.common.eventbus.EventBus.post(EventBus.java:275) [guava-16.0.jar:?]
at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:118) [LoadController.class:?]
at cpw.mods.fml.common.Loader.initializeMods(Loader.java:691) [Loader.class:?]
at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:288) [FMLClientHandler.class:?]
at com.creativemd.minecraftloader.ClientHelper.startGame(ClientHelper.java:307) [ClientHelper.class:?]
at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java) [bao.class:?]
at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:867) [bao.class:?]
at net.minecraft.client.main.Main.main(SourceFile:148) [Main.class:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0_67]
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.11.jar:?]
at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.11.jar:?]
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0_67]
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.7.0_67]
at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.7.0_67]
at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:286) [NewLaunch.jar:?]
at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:376) [NewLaunch.jar:?]
at org.multimc.EntryPoint.listen(EntryPoint.java:165) [NewLaunch.jar:?]
at org.multimc.EntryPoint.main(EntryPoint.java:54) [NewLaunch.jar:?]
```

**Comment by bdew** [comment]
This is with BC 6.1 i assume? 

The current version of Gendustry is built against 6.0.x, i will switch to 6.1 for future versions.

This should cause any problems except assembly table recipes for my processors being unavailable, there should still be normal crafting recipes for them.

**Comment by Painstouch** [comment]
Yeah, I am using BC 6.1.4.
Thanks for the effort!
EDIT: Note, the instance did launch successfully, not tested in-game compatibility though.

**Comment by bdew** [comment]
fixed in 5278bc2752fceec6ad0fe77c680014e863bc7f4f
